### PR TITLE
TikTok Ads Offline Events documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Added documentation for TikTok Ads Offline Events destination

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

**Review by sandeepdsvs — APPROVED** [review]

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
+---
+title: "TikTok Ads Offline Events"
+description: Send your event data from RudderStack to TikTok Ads Offline Events.
+---
+
+[TikTok Ads](https://www.tiktok.com/business/en/apps/tiktok?tt4b_lang_redirect=1") is TikTok's online advertising platform. TikTok Ads Offline Events lets you track and send your conversions data after a user clicks your TikTok ad but does not directly proceed to sale in the online mode.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:6]: I understand that:
* TikTok Ads is TikTok's **online** advertising platform. 
* TikTok Ads **Offline** Events occur when a user click TikTok ad but does not directly proceed to sale in the **online** mode.

But this **online** and **offline** mentions simultaneously can be lil confusing. How about we skip first line and just explain TikTok Ads Offline Events?

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+Find the open source code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/src/v0/destinations/tiktok_ads_offline_events">GitHub repository</a>.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:8]:
```suggestion
Find the open source transformer code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/src/v0/destinations/tiktok_ads_offline_events">GitHub repository</a>.
```

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **TikTok Ads Offline Events**.
+2. Assign a name to the destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure TikTok Ads Offline Events as a destination, you will need to configure the following settings:
+
+- **Access Token**: Enter your TikTok access token (also called Long Term Access Token) by following the steps mentioned in the [Authorization](https://ads.tiktok.com/marketing_api/docs?id=1738373141733378) and [Authentication](https://ads.tiktok.com/marketing_api/docs?id=1738373164380162) sections of the TikTok documentation.
+- **Mapping to trigger the TikTok Ads Offline standard events for the respective events**: Enter the **Event Name** and select the corresponding **Standard Event** from the dropdown to be triggered when that event is called. You can specify multiple standard events for one event name and vice versa.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:29]:
```suggestion
- **Mapping to trigger the TikTok Ads Offline standard events for the respective events**: Enter the **Event Name** and select the corresponding **Standard Event** from the dropdown you want to trigger when that event is called. You can specify multiple standard events for one event name and vice versa.
```

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+- **Hash Contextual User Properties (SHA-256)**: Use this setting to hash the user properties like email and phone using SHA-256 encryption.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:30]:
```suggestion
- **Hash Contextual User Properties (SHA-256)**: Use this setting to hash the user properties like `email` and `phone` using SHA-256 encryption.
```

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+      quantity: 2,
+      category: "Games",
+    },
+  ],
+})
+```
+
+### Standard events
+
+To send a `track` event as a standard TikTok Ads Offline Event, you must specify the event mapping in the RudderStack  dashboard setting, as shown:
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:70]:
```suggestion
To send a `track` event as a standard TikTok Ads Offline Event, you must specify the event mapping in the RudderStack dashboard's <Link to="#connection-settings">connection setting</Link>, as shown: 
```

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+})
+```
+
+### Standard events
+
+To send a `track` event as a standard TikTok Ads Offline Event, you must specify the event mapping in the RudderStack  dashboard setting, as shown: 
+
+<img src="../../assets/event-stream-destinations/tiktok-ads-offline-standard-events.png" alt="TikTok Ads Offline standard events" />
+
+For more information on this setting, refer to the <Link to="#connection-settings">Connection settings</Link> section.
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:74]:
```suggestion
```

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx (hunk 1 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/tiktok-ads-offline-events.mdx:6]: Makes sense.

**Pull request merged**